.env

```diff
@@ -0,0 +1,5 @@
+PORT=3001
+
+WEATHER_API_KEY=6111091c47f743c7bc1410fabffbbb88
+
+MOVIE_API_KEY=637d69539e64f605b5d28ed0a9519e57
```

Comment:
Hi there!

I know you are still working on lab. Feedback here is to remove your .env from git. Grab a TA if you can to ensure that this is properly ignored (if you have troubles) 👍